mbs-s0015-20240504/make_masks.py

```diff
@@ -0,0 +1,73 @@
+import os
+
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy.stats import median_abs_deviation
+from mpi4py import MPI
+from pixell import enmap, enplot
+
+comm = MPI.COMM_WORLD
+opj = os.path.join
+
+idir = '/mnt/home/aduivenvoorden/project/so/20240504_mss0002/RC1.r01'
+odir = '/mnt/home/aduivenvoorden/project/so/20240504_mss0002_lat/masks'
```

Comment:
path fix here as well

Reply:
Same as above. Added NERSC path and clarified that these are CCA cluster paths.